### PR TITLE
Upgrades for LSG release v2405.9.2

### PR DESCRIPTION
**Pull request by dallasd1**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] If you are adding/removing a .spec file that has multiple-versions supported, please add [@microsoft/cbl-mariner-multi-package-reviewers](https://github.com/orgs/microsoft/teams/cbl-mariner-multi-package-reviewers) team as reviewer [(Eg. golang has 2 versions 1.18, 1.21+)](https://github.com/microsoft/azurelinux/tree/2.0/SPECS/golang)
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES/NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: SPECS/cloud-hypervisor/cloud-hypervisor.signatures.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-  "cloud-hypervisor-32.0-cargo.tar.gz": "2dd7ca374109ba337afeb0ff95d5edac8193431ec74cdbb6b1a400c600f4d915",
-  "cloud-hypervisor-32.0.tar.gz": "b9754a5ecd26697e5416a642345b2f35f4fdc983a83d540d740978309f2eb419",
-  "config.toml": "6d2aeec19782ae17eb2708262b0a7c551db3cc36b56542abca18d577de042458"
- }
-}
\ No newline at end of file
+  "Signatures": {
+    "config.toml": "6d2aeec19782ae17eb2708262b0a7c551db3cc36b56542abca18d577de042458",
+    "cloud-hypervisor-38.0.72.tar.gz": "e6d15d99c5d9ec4bede43ef8fac971d2cc0ae49a7eafffc6ca7e5b948ed4282a",
+    "cloud-hypervisor-38.0.72-vendor.tar.gz": "505887d56fb2a5ed3e19ccba42552553f9513720a0601e6ce74429d71ea53cb2"
```
Comment (Redent0r) [inline SPECS/cloud-hypervisor/cloud-hypervisor.signatures.json:5]: This has been renamed from `-cargo` to `-vendor`, should also be renamed in SPEC file

**Comment by Redent0r** [comment]
todo: add cloud-hypervisor-cvm

File: SPECS/cloud-hypervisor-cvm/fetch-private-deps [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```
Comment (ms-mahuber) [inline SPECS/cloud-hypervisor-cvm/fetch-private-deps:1]: just remind me, what is this for again?

File: SPECS/cloud-hypervisor-cvm/update_src [hunk 3]
```diff
@@ -0,0 +1,72 @@
+#/bin/bash
```
Comment (ms-mahuber) [inline SPECS/cloud-hypervisor-cvm/update_src:1]: Just remind me, what is this for?

File: SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec [hunk 4]
```diff
@@ -0,0 +1,226 @@
+# This spec file assumes you're building on an environment where:
```
Comment (ms-mahuber) [inline SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec:1]: General thought: Let's try to align this as much as possible with the cloud-hypervisor proper package.

Comment (ms-mahuber) [inline SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec:1]: Also one reminder: Any PRs we make to main, please open a PR for 3.0-dev

File: SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+
+%if ! 0%{?using_rustup}
+BuildRequires:  rust >= 1.62.0
+BuildRequires:  cargo >= 1.62.0
+%endif
+
+Requires: bash
+Requires: glibc
+Requires: libgcc
+Requires: libcap
```
Comment (ms-mahuber) [inline SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec:48]: I think we should add a Conflicts with cloud-hypervisor.

We should then install cloud-hypervisor-cvm to the same places where cloud-hypervisor gets installed to.

Also, please consider changes we may need to make to the kata-containers and/or kata-containers-cvm package sources or SPEC files. We will need to change the paths to cloud-hypervisor.

**Comment by Redent0r** [comment]
Closing in favor of https://github.com/microsoft/azurelinux/pull/9161

**Pull request closed without merging**